dashboard/test/dsl/script_dsl_test.rb

```diff
@@ -718,4 +718,34 @@ def setup
     output, _ = ScriptDSL.parse(input_dsl, 'test.script', 'test')
     assert_equal expected, output
   end
+
+  test 'script DSL with single quotes' do
+    input_dsl = <<-DSL.gsub(/^\s+/, '')
```

Comment:
I think you could just use [squiggly heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) since we're on Ruby 2.5.0.

Reply:
TIL!